types/react-better-password/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for react-better-password 1.0
+// Project: https://github.com/karaggeorge/react-better-password
+// Definitions by: Mary Huynh <https://github.com/mhuynh1>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+import * as React from 'react';
+export interface PasswordProps {
+    className?: string;
+    mask?: '•' | string;
+    onChange: any;
```

Comment:
Might you consider writing this as `(value: string) => void` instead of `any` (and marking it as [optional](https://github.com/karaggeorge/react-better-password/blob/master/src/index.js#L54) like the rest)? Or does it have a much different type than that?